data/hai/hai news.txt

```diff
@@ -396,6 +396,73 @@ news "hai advertisement in Hai space"
 		word
 			`"`
 
+news "hai attacked by pirates"
+	location
+		government "Hai"
+	to show
+		random < 1
+	name
+		word
+			"Hai civilian"
+			"Hai youth"
+	message
+		word
+			`"`
+		word
+			"The human ship carrying my family was attacked by pirates when they came back from human space. I guess I may as well be joining the Unfettered now, I have nothing to lose."
+			"Pirates destroyed the human ship my children were on. They were always curious about the humans, and look what that got them! I will join the Unfettered; they are right."
+			"Human ships are so frail... My sister was on one, and she told me it lost all of his shields in a few shots! Thankfully, human weapons are not that strong either, so they were able to escape."
```

Comment:
You mean say it was the pilot's skill that allowed her escape, and that the sister is better than the one speaking to you?

Reply:
In my interpretation, your sentence means `the ship lost all of the ship's shields`. I have no idea who `he` could be in your sentence, since ships (and spaceships) are generally treated as female in English. So it is either `it lost all of its shields` (as an object) or `she lost all of her shields`.

The second sentence is fine. When I said `she/her would be even better`, I was referring to the pronouns of the spaceship.